iceberg_rust_ffi/Cargo.toml

```diff
@@ -12,14 +12,14 @@ default = ["julia"]
 julia = []
 
 [dependencies]
-iceberg = { git = "https://github.com/RelationalAI/iceberg-rust.git", rev = "ae83309fd198ec30f052a7e9f983711c5f581aea" }
+iceberg = { git = "https://github.com/RelationalAI/iceberg-rust.git", rev = "5003e5d01cc081fbaeaedb3e4b9321c7727c5ae3" }
```

Comment:
should update sha to point to main once https://github.com/RelationalAI/iceberg-rust/pull/14 is merged